stash-merritt/lib/stash/merritt/repository.rb

```diff
@@ -9,7 +9,7 @@ class Repository < Stash::Repo::Repository
 
       ARK_PATTERN = %r{ark:/[a-z0-9]+/[a-z0-9]+}
 
-      def initialize(url_helpers:)
+      def initialize(url_helpers:, threads: 1)
```

Comment:
Should this instead refer to the new config file entry rather than a hard-coded value?

Reply:
Good catch, though it seems like the concurrency library doesn't work the same on Travis as locally.  :-/  IDK, maybe if I make it sleep for a while or do something else it might do what it is supposed to.  I'll see what I can figure out.